doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md

```diff
@@ -34,14 +34,23 @@
     + [2.2.2 Checkpoint](#222-checkpoint)
       - [Stage-1 Get current ConfigDB JSON config](#stage-1-get-current-configdb-json-config)
       - [Stage-2 Save JSON config](#stage-2-save-json-config)
-    + [2.2.3 rollback](#223-rollback)
+    + [2.2.3 Rollback](#223-rollback)
       - [Stage-1 Get current ConfigDB JSON config](#stage-1-get-current-configdb-json-config-1)
       - [Stage-2 Get checkpoint JSON config](#stage-2-get-checkpoint-json-config)
       - [Stage-3 Generate the diff as JsonPatch between current config and checkpoint](#stage-3-generate-the-diff-as-jsonpatch-between-current-config-and-checkpoint)
       - [Stage-4 Apply-Patch](#stage-4-apply-patch)
       - [Stage-5 Verify config rollback](#stage-5-verify-config-rollback)
       - [Fail-safe Action](#fail-safe-action-1)
       - [Logging](#logging-1)
+    + [2.2.4 Replace](#224-replace)
+      - [Stage-1 Get current ConfigDB JSON config](#stage-1-get-current-configdb-json-config-2)
```

Comment:
Here the stages refer to the stage of development?

Reply:
the stages are general explanation of how it should be developed